domain/src/main/java/org/fao/geonet/domain/GeonetEntity.java

```diff
@@ -183,7 +183,7 @@ protected static boolean isNotLabel(final String descName) {
      * @param descName
      * @return
      */
-    protected static boolean isNotADuplicatedMethodWithAnotherReturnType(final String descName) {
+    protected static boolean isADuplicatedMethodWithAnotherReturnType(final String descName) {
```

Comment:
I see a rename that change the semantics, but the method content looks the same. Previous name was not semantically correct?

Reply:
Same as before. It was confusing.